ext/GeometryOpsLibGEOSExt/buffer.jl

```diff
@@ -0,0 +1,11 @@
+function GO.buffer(::GEOS, geom, distance)
+    return LG.buffer(GI.convert(LG, geom), distance)
+end
+
+function GO.buffer(::GEOS{(:cap_style, :join_style)}, geom, distance)
```

Comment:
What if users only pass `cap_style`?

Reply:
Yet to be implemented :D - one of the things I haven't figured out yet is dispatch on subsets. Maybe we don't dispatch on keywords and test them internally instead?

Reply:
Yeah dispatch on keywords might not scale well, probably a conditional is easier.